p2pserver/actor/req/consensus.go

```diff
@@ -27,3 +27,9 @@ var ConsensusPid *actor.PID
 func SetConsensusPid(conPid *actor.PID) {
 	ConsensusPid = conPid
 }
+
+func NotifyEmergencyGovCmd(cmd interface{}) {
```

Comment:
if we change to restart consensus module, this still need?

Reply:
Yes, it is a wrapped interface to trigger restarting consensus module, since p2p and consensus use actor to communicate.